golang.mk

```diff
@@ -34,4 +34,11 @@ fmt:
 	@echo "==> Formatting source"
 	@gofmt -s -w .
 	@echo "==> Done"
-.PHONY: fmt
\ No newline at end of file
+.PHONY: fmt
+
+# Format all files and fix imports
+imports:
+	@echo "==> Formatting source"
+	@goimports -w .
```

Comment:
This will do your vendor folder as well.

Reply:
Good catch. `$(shell go list -f {{.Dir}} ./... | grep -v /vendor/)` did the trick